### PR TITLE
Disambiguate readme wording

### PR DESCRIPTION
**Pull request by jil-stripe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
 
-**Note**: To be
+**Note**: This package dynamically loads the Stripe Terminal SDK from 
+`https://js.stripe.com` and wraps the SDK's global`StripeTerminal` function. 
+To be
 [PCI compliant](https://stripe.com/docs/security#validating-pci-compliance), you
-must load the SDK directly from `https://js.stripe.com`. You cannot include it
-in a bundle or host it yourself. This package wraps the global `StripeTerminal`
-function provided by the the SDK script as an ES module.
+must load the SDK directly from `https://js.stripe.com` using this library or by
```
Comment (jdivock-stripe) [inline README.md:10]: > must load the SDK directly from `https://js.stripe.com` *by* using this library or by loading it…


I also think we don't want to present using a `<script/>` tag as an equal option. We support it as documented elsewhere in the README, but I think it'll cause confusion to suggest it here. What do you think?

Comment (jil-stripe) [reply README.md:10]: That's a good point. I don't think we've nailed the wording yet.

**Comment by jdivock-stripe** [comment]
Looks good, but the repo is configured to run prettier on markdown so it's failing out. Should just have to run the yarn target and re-push

**Comment by jil-stripe** [comment]
That's what I get for writing in the Github interface.

**Pull request merged**